src/istgt_lu_disk.c

```diff
@@ -952,15 +954,15 @@ istgt_lu_disk_init(ISTGT_Ptr istgt __attribute__((__unused__)), ISTGT_LU_Ptr lu)
 		rc = pthread_cond_init(&spec->cmd_queue_cond, NULL);
 		if (rc != 0) {
 			ISTGT_ERRLOG("LU%d: cond_init() failed errno:%d\n", lu->num, errno);
-			return -1;
```

Comment:
lets avoid error handling in this function, and, just return -1

Reply:
ok